topics/phylogenetics/tutorials/MTB phylogeny/tutorial.md

```diff
@@ -0,0 +1,364 @@
+These trees look rather different at a first glance, but they are identical except for one key aspect: tree A is **rooted** while tree B is not.
+
+What is the difference between the two? First, let's state what is the same in the two trees: the tree **topology**, that is, the relative branching order. The same groupings are present in the two trees: they both contain the same information about the **relatedness** of strains. An example: TB isolated from Peruvian mummies is most similar to *M. pinnipedii* known from marine mammals; they share a most recent common ancestor. This can be seen in the rooted as well as in the unrooted tree.
+
+The key difference between the rooted and the unrooted tree is that only the rooted tree shows the **direction** and sequence of branching events. The unrooted tree does not tell us, for example, whether *M. bovis* diverged early or late in the history of the MTBC. It is thus compatible with the old hypothesis that human TB evolved from animal TB. The rooted tree shows that this hypothesis is most likely wrong: animal-associated strains are not ancestral to human-associated strains.
+
+The best way to root a tree is by including an **outgroup**: a species or lineage which we know *a priori* to lay outside the phylogeny we're interested in. *M. canetti* usually serves this purpose for studying the MTBC, but you can also root, for example, a phylogeny of lineage 2 by including a lineage 4 strain.
+
+<img src="./images/mtbc_1strainPerLineage.nwk.COMB.svg" alt="drawing" width="800"/>
+<figcaption align = "left">Fig.1 - Rooted (A) and unrooted (B) phylogeny of the MTBC. Some basic tree vocabulary is shown in red.</figcaption>
+
+
+## Branch lengths
+Besides relatedness and direction, a third important piece of information contained in a phylogeny is the branch length. When a phylogeny was estimated from DNA or protein sequences, branch length usually reflects the evolutionary distance between nodes in the tree. This information can be used to translate distance in terms of expected nucleotide changes into years, and thus to connect evolutionary change to historical events. Molecular dating is not covered in this tutorial, but if you should be interested in this topic, here is a [recent study](https://doi.org/10.1371/journal.ppat.1008067) showing the promises and problems of molecular dating with MTB.
+
+As branch lengths reflect evolutionary distances, they can also be used to identify transmission clusters and outbreaks. Below is a (rooted) tree of the Central Asian Clade (CAC), which is part of lineage 2 [(Eldholm et al. 2016)](https://doi.org/10.1073/pnas.1611283113). The orange color highlights the Afghan strain family within the CAC. At the bottom of the tree, note the clade with short branch lengths. This is how one would expect an outbreak to look in a phylogenetic tree: a set of strains clustering together and separated by extremely short branches, reflecting their almost identical genomes.
```

Comment:
For all of these papers you reference, they could be better done like this:

```suggestion
As branch lengths reflect evolutionary distances, they can also be used to identify transmission clusters and outbreaks. Below is a (rooted) tree of the Central Asian Clade (CAC), which is part of lineage 2 ({% cite eldholm2016 %}). The orange color highlights the Afghan strain family within the CAC. At the bottom of the tree, note the clade with short branch lengths. This is how one would expect an outbreak to look in a phylogenetic tree: a set of strains clustering together and separated by extremely short branches, reflecting their almost identical genomes.
```

and then use a service like doi2bib.org to use convert the reference into a bibtex format and use in the `tutorial.bib` file

Reply:
This was changed. I couldn't preview the citations, so I don't know how they will be formatted.